litle.yml

```diff
@@ -1,15 +1,15 @@
 :litle:
   - :account_id: "USD"
     :merchant_id: "your-merchant-id-USD"
-    :username: "your-username"
+    :login: "your-username"
```

Comment:
You could switch to `ENV` variables here, similar to what is done in [CyberSource](https://github.com/killbill/killbill-cybersource-plugin/blob/master/cybersource.yml#L4) for instance.

That way, no need for any changes in tests?